util.go

```diff
@@ -4,12 +4,19 @@ import (
 	"crypto/tls"
 	"net/http"
 	"os"
+	"strings"
 	"time"
 )
 
 func BreakArgsBySeparator() (left, right []string) {
 	seenSeparator := false
 	for _, param := range os.Args[1:] {
+		if strings.HasPrefix(param, "-watch") ||
+			strings.HasPrefix(param, "--watch") ||
+			strings.HasPrefix(param, "-filter") ||
+			strings.HasPrefix(param, "--filter") {
+			continue
+		}
```

Comment:
Not sure if there's a better way of doing this. The problem is that the jump argument keywords are otherwise taken as SSH arguments which then breaks the SSH command as it tries to do e.g. `ssh -filter`.